elliottlib/bzutil.py

```diff
@@ -606,8 +607,11 @@ def remove_bugs(self, advisory_obj, bugids: List, noop=False):
         if noop:
             print(f"Would've removed bugs: {bugids}")
             return
-        advisory_id = advisory_obj.errata_id
-        return errata.remove_multi_jira_issues(advisory_id, bugids)
+        try:
+            advisory_obj.removeJIRAIssues(bugids)
+            advisory_obj.commit()
+        except ErrataException as e:
+            raise e
```

Comment:
Why have this in a `try` block if the `except` just reraises the exception?

The bugzilla code still does a `return`. Any reason to drop that here?

Reply:
I don't think we need `return` here, if no exception raised it means success

Reply:
Think there's value in having the function signature equal between the two bugzilla and jira implementations.

Reply:
do you mean the function `remove_bugs` for bugzilla tracker? https://github.com/openshift/elliott/blob/master/elliottlib/bzutil.py#L704 it's using post api to implement it so we need the response, errata_tool have the function `removeBugs` we can also use errata_tool for it https://github.com/red-hat-storage/errata-tool/blob/master/errata_tool/erratum.py#L616